docs/user/lib_getting_started.md

```diff
@@ -0,0 +1,321 @@
+# Getting Started with the Library
+
+There are two ways to get started with pyntc.
+
+The first way is to use the `ntc_device` object. Just pass in all required parameters to the object to initialize your device.  Here we are showing the import, but renaming the object to `NTC`.
+
+```
+>>> from pyntc import ntc_device as NTC
+>>>
+```
+
+Like many libraries, we need to pass in the host/IP and credentials.  Because this is a multi-vendor/API library, we also use the `device_type` parameter to identify which device we are building an instance of.
+
+pyntc currently supports four device types:
+* cisco_aireos_ssh
+* cisco_asa_ssh
+* cisco_ios_ssh
+* cisco_nxos_nxapi
+* arista_eos_eapi
+* juniper_junos_netconf
+* f5_tmos_icontrol
+
+The example below shows how to build a device object when working with a Cisco IOS router.
+
+```
+>>> # CREATE DEVICE OBJECT FOR AN IOS DEVICE
+>>>
+>>> csr1 = NTC(host='csr1', username='ntc', password='ntc123', device_type='cisco_ios_ssh')
+>>>
+```
+
+And here is an object for a Cisco Nexus device:
+
+```
+>>> # CREATE DEVICE OBJECT FOR A NEXUS DEVICE
+>>>
+>>> nxs1 = NTC(host='nxos-spine1', username='ntc', password='ntc123', device_type='cisco_nxos_nxapi')
+>>>
+```
+
+The second way to get started with pyntc is to use the pyntc configuration file.  This was modeled after Arista's `.eapi.conf` file.  Our file is called `.ntc.conf`
+
+This simplifies creating device objects since you no longer need to specify credentials and other device specific parameters when you build the device object.  Instead, they are stored in the conf file.
+
+
+# pyntc Configration File
+
+- filename:  `.ntc.conf`
+- Priority of locating the conf file:
+  - `filename` param in `ntc_device_by_name`
+  - Environment Variable aka `PYNTC_CONF`
+  - Home directory `.ntc.conf`
+- Specify device_type and a name
+- host is not required if the name is the device's FQDN
+- Four supported device types: `cisco_nxos_nxapi`, `cisco_ios_ssh`, `arista_eos_eapi`, and `juniper_junos_netconf`
+
+Here is an example `.ntc.conf` file:
+
+```bash
+[cisco_nxos_nxapi:nxos-spine1]
+host: 31.220.64.117
+username: ntc
+password: ntc123
+transport: http
+
+[cisco_ios_ssh:csr1]
+host: 176.126.88.94
+username: ntc
+password: ntc123
+port: 22
+
+[juniper_junos_netconf:vmx1]
+host: 176.126.88.99
+username: ntc
+password: ntc123
+
+```
+
+We can now build device objects just by referencing the name of the device from the conf file.
+
+```
+>>> from pyntc import ntc_device_by_name as NTCNAME
+>>>
+>>> csr1 = NTCNAME('csr1')
+>>>
+>>> nxs1 = NTCNAME('nxos-spine1')
+>>>
+>>> vmx1 = NTCNAME('vmx1')
+```
+
+
+Once the device object is creating using either `ntc_device` or `ntc_device_by_name`, you can start using the built-in device methods in pyntc.
+
+Note: the only method and property not supported on all devices is `install_os`.  It is not supported on Juniper Junos devices.
+
+### Gathering Facts
+
+- Use `facts` device property
+
+On a Nexus device:
+
+```
```

Comment:
```suggestion
```python
```

Reply:
there is a bunch of these :) can you knock them out?

Reply:
thanks, just copy/pasted from old readme, will get them all updated.